remo/featuredrep/forms.py

```diff
@@ -13,7 +16,13 @@ class FeaturedRepForm(happyforms.ModelForm):
             userprofile__registration_complete=True,
             groups__name='Rep').order_by('first_name'))
 
+    def clean_created_on(self):
+        if self.cleaned_data['created_on'] > now() + datetime.timedelta(days=30):
```

Comment:
this fails if the user didn't add any date (leave the whole form empty to test):

![screen shot 2018-04-14 at 17 09 04](https://user-images.githubusercontent.com/330324/38769972-ef20b884-400b-11e8-8e81-eb2ec0786769.png)

Reply:
Cool, so I have to add a check if isset.